File: Core/Workspace/Typescript.legacy/Site/wwwroot/Core/Workspace/Data/Requests/InvokeRequest.ts
namespace Allors.Data {

    export interface InvokeOptions {
        // Isolated
        i?: boolean;

        // ContinueOnError
        c?: boolean;
    }

    export interface Invocation {
        i: string;
        v: string;
        m: string;
    }

    export interface InvokeRequest {
        i: Invocation[];
        o?: InvokeOptions;
    }
}

File: dotnet/system/protocol/allors.protocol.json/api/invoke/invokerequest.cs
// <copyright file="InvokeRequest.cs" company="Allors bvba">
// Copyright (c) Allors bvba. All rights reserved.
// Licensed under the LGPL license. See LICENSE file in the project root for full license information.
// </copyright>

namespace Allors.Protocol.Json.Api.Invoke
{
    public class InvokeRequest
    {
        /// <summary>
        ///  List of Invocations
        /// </summary>
        public Invocation[] l { get; set; }

        /// <summary>
        /// Options
        /// </summary>
        public InvokeOptions o { get; set; }
    }
}
